monkey/infection_monkey/credential_collectors/credential_components/ntlm_hash.py

```diff
@@ -0,0 +1,8 @@
+from ..credential_types import CredentialTypes
+
+from .i_credential_component import ICredentialComponent
+
+
+class NtlmHash(ICredentialComponent):
```

Comment:
I renamed this to LMHashes, it now contains ntlm_hash and lm_hash

Reply:
I was thinking about combining them into a single credential component, since they represent the same password. Not sure what's a good name though. I'll probably split them up, but we shouldn't do it, as it will lead to component explosion